sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -279,6 +275,25 @@ def to_table(self) -> pyarrow.Table:
         return self.client.query(self.query).to_arrow()
 
 
+def block_until_done(client, bq_job):
+    def _is_done(job_id):
+        return client.get_job(job_id).state in ["PENDING", "RUNNING"]
+
+    @retry(wait=wait_fixed(10), stop=stop_after_delay(1800), reraise=True)
+    def _wait_till_done(job_id):
```

Comment:
```suggestion
    def _wait_until_done(job_id):
```
Till is technically correct, but I'd prefer to stick to until since it's more commonly used.

Reply:
Told you English was not my primary language 😉